ethereum/p2p/src/main/java/org/hyperledger/besu/ethereum/p2p/discovery/PeerDiscoveryAgent.java

```diff
@@ -315,6 +315,12 @@ protected void handleOutgoingPacket(final DiscoveryPeer peer, final Packet packe
                 if (err instanceof SocketException && err.getMessage().contains("unreachable")) {
                   LOG.debug(
                       "Peer {} is unreachable, packet: {}", peer, wrapBuffer(packet.encode()), err);
+                } else if (err instanceof SocketException
+                    && err.getMessage().contains("permitted")) {
```

Comment:
Can / should we make that a bit more precise than "permitted"?

Reply:
I'm going off the previous one which searches for "unreachable" 😛 but yeah I'll put in the whole message there